android/java/org/chromium/chrome/browser/shields/FilterListServiceFactory.java

```diff
@@ -0,0 +1,63 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
please use correct header, 2023 and https

Reply:
updated